packages/block-library/src/media-text/index.php

```diff
@@ -29,28 +29,76 @@ function render_block_core_media_text( $attributes, $content ) {
 		return $content;
 	}
 
-	$image_tag = '<figure class="wp-block-media-text__media"><img>';
-	$content   = preg_replace( '/<figure\s+class="wp-block-media-text__media">/', $image_tag, $content );
+	$has_media_on_right    = isset( $attributes['mediaPosition'] ) && 'right' === $attributes['mediaPosition'];
+	$image_fill            = isset( $attributes['imageFill'] ) && $attributes['imageFill'];
+	$focal_point           = isset( $attributes['focalPoint'] ) ? round( $attributes['focalPoint']['x'] * 100 ) . '% ' . round( $attributes['focalPoint']['y'] * 100 ) . '%' : '50% 50%';
+	$media_size_slug       = isset( $attributes['mediaSizeSlug'] ) ? $attributes['mediaSizeSlug'] : 'full';
+	$unique_id             = 'wp-block-media-text__media-' . wp_unique_id();
+	$image_tag             = '<img class="wp-block-media-text__featured_image">';
```

Comment:
This class name is probably not needed, and does not exist in the editor, which makes it difficult to use it for styling anyway.

Reply:
I forgot that I already removed it from the resulting markup, it is _only_ used for identifying the image tag.

Reply:
Do you need it at all?

Wouldn't we expect an image to come after the figure? E.g., 

```php
		$image_tag_processor->next_tag(
			array(
				'tag_name'   => 'img',
		);
```

Otherwise you could delete it as you do with the id. Probably best to remove if it's not required 👍🏻